packages/jest-config/src/normalize.js

```diff
@@ -288,6 +289,13 @@ const normalizeRootDir = (options: InitialOptions): InitialOptions => {
     );
   }
   options.rootDir = path.normalize(options.rootDir);
+
+  try {
+    options.rootDir = realpath(options.rootDir);
+  } catch (e) {
```

Comment:
it threw on CI since we have some `/root/path` in tests, which got a permission error